CONTRIBUTIONS.md

```diff
@@ -27,4 +28,19 @@ Check if the issue has been addressed or is in progress and if not , only then d
 Describe the enhancement in mind and what you would expect to have resulted from this process. Submit the enhancement with the `enhancement` tag.
 
 ## Pull requests
-Reference the issue or enhancement being referenced in the pull request and submit the pull request to the `development` branch.
\ No newline at end of file
+Reference the issue or enhancement being referenced in the pull request and submit the pull request to the `development` branch.
+
+## Workflow
+We use the git workflow on our projects. If you are unfamiliar with the workflow, here is an awesome primer by [Atlassian](https://www.atlassian.com/git/tutorials/comparing-workflows/gitflow-workflow). Basically, we have a `master` branch that serves for official release cycles, a `develop` branch that sums up features for the next release. Each `feature/chore/bug-fix` needs to be in its own branch originating from the  `develop` branch. When the develop branch has acquired enough features for a `release`, a temporary branch is created (intuitively named **release**) to mark the next release cycle. At this point, **No new** features will be allowed, just bug fixes or documentations. When we are confident enough and after extensive testing, the new branch is merged to the `master` with a version number, it is merged into the `develop` branch to get the fixes or modifications made on the `release` branch. After all that, the branch is deleted and we go on and start working on features for the next release (from the `develop` branch).
+
+## API Documentation Style Guide
```

Comment:
Hi @yegow, great work on the docs! The team decided to move forward with swagger instead of apiary because of the completely free and open swagger ui that swagger offers which comes with a sandbox and documentation element. So sorry you have to remove this bit, I'll gladly merge in the rest after you make the changes.